handler/src/main/java/io/netty/handler/pcap/PcapWriteHandler.java

```diff
@@ -114,8 +114,10 @@
      * {@code writePcapGlobalHeader} is set to {@code true}.
      *
      * @param outputStream OutputStream where Pcap data will be written
+     * @throws NullPointerException If {@link OutputStream} is {@code null} then we'll throw an
+     *                              {@link NullPointerException}
      */
-    public PcapWriteHandler(OutputStream outputStream) {
+    public PcapWriteHandler(OutputStream outputStream) throws NullPointerException {
```

Comment:
Harmless, but no need to declare NPE on the signature; just declaring in the javadoc is good enough.